### PR TITLE
Initial version of the connector

### PR DESCRIPTION
**Pull request by LeeVi3w**
Initial version of the connector which supports querying CSV Files and database tables, with S3 and MySql connections. Some limitations may exist which will be fixed with a more well rounded mapping between the 2 technologies.

### CODE REVIEW

File: src/main/java/org/fybrik/datacatalog/api/GetAssetInfoApiController.java [hunk 1]
```diff
@@ -1,129 +1,90 @@
...
-    mapper.setSerializationInclusion(Include.NON_NULL);
-
-    String jsonString;
-    try {
-      jsonString = mapper.writeValueAsString(response);
-    } catch (Exception e) {
-      e.printStackTrace();
-      return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR);
+
+    private static final String SERVER_NAME = "omas-server";
```
Comment (cdoron) [inline src/main/java/org/fybrik/datacatalog/api/GetAssetInfoApiController.java:30]: I assume that these hard-coded values are temporary, right?

File: src/main/org/odpi/openmetadata/egeria/mapping/assets/RelationalTableTypeMapper.java [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+     * @param requestedEntity Egeria RelationalTable entity represented as a DataAssetElement
+     *
+     * @return List of ResourceColumn objects
+     *
+     * @throws InvalidParameterException if a parameter validation fails
+     * @throws PropertyServerException if an error occurs while serving the request
+     * @throws UserNotAuthorizedException if the user is not authorized to make the request
+     */
+    private List<ResourceColumn> getColumns(AssetCatalogBean requestedEntity) throws InvalidParameterException, PropertyServerException, UserNotAuthorizedException {
+        List<SchemaAttributeElement> nestedSchemaAttributes = dataAssetExchangeClient.getNestedSchemaAttributes(userId,
```
Comment (cdoron) [inline src/main/org/odpi/openmetadata/egeria/mapping/assets/RelationalTableTypeMapper.java:150]: What do the parameters below stand for?

File: src/main/java/org/fybrik/datacatalog/api/GetAssetInfoApiController.java (hunk 1 above)
Comment (LeeVi3w) [reply src/main/java/org/fybrik/datacatalog/api/GetAssetInfoApiController.java:30]: Yes, these are temporary. They should be changed accordingly, as I've not made them into a config yet.

File: src/main/org/odpi/openmetadata/egeria/mapping/assets/RelationalTableTypeMapper.java (hunk 2 above)
Comment (cdoron) [reply src/main/org/odpi/openmetadata/egeria/mapping/assets/RelationalTableTypeMapper.java:150]: please add comments explaining the parameters

**Review by cdoron — APPROVED** [review]

**Comment by cdoron** [comment]
Future steps:
1. add automatic java style checker
2. update README.md
3. document how to set up Egeria

**Pull request merged**